test/EFCore.PG.Tests/NpgsqlMigrationBuilderTest.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Xunit;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL
+{
+    public class NpgsqlMigrationBuilderTest
+    {
+        [Fact]
+        public void IsNpgsql_when_using_EfCorePostgreSql()
```

Comment:
Let's rename this closer to the [upstream equivalent](https://github.com/aspnet/EntityFrameworkCore/blob/c4705ffc77ae204ee4c9111eb6314a895736cd62/test/EFCore.SqlServer.Tests/SqlServerMigrationBuilderTest.cs#L12):

`IsSqlServer_when_using_SqlServer` => `IsNpgsql_when_using_Npgsql`

Reply:
Renamed